vhdbuilder/release-notes/autonotes/main.go

```diff
@@ -80,8 +88,33 @@ func run(ctx context.Context, cancel context.CancelFunc, fl *flags) []error {
 
 	enforceInclude := len(include) > 0
 
+	// Get windows base image versions frpm the updated windows-image.env
+	var wsImageVersionFilePath = filepath.Join("vhdbuilder", "packer", "windows-image.env")
```

Comment:
It seems like that you are updating an existing file. Will it block Linux?

Reply:
No. this `variable` is only used once below in line `93`. And this is just for building the map not updating.

Reply:
I don't think it will block Linux because this file will be called by line 28, https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/scripts/automate_release_notes.sh. The root path is AgentBaker so I don't understand why wouldn't the file `vhdbuilder/packer/windows-image.env` exist?

Reply:
I think this should be fine, I double checked that the file windows-image.env actually exists, and yes, we run autonotes from root directory, so it should be alright. 

To confirm, you can take any weekly linux release and run autonotes with your changes to make sure it works for linux as well, but I think it will.

Reply:
Here's the command I ran `autonotes --build 78770231 --ignore 2019-containerd,2022-containerd,2022-containerd-gen2,marinerv2-gen2-kata-trustedlaunch` with my changes and below is the output. Should be fine. (_`marinerv2-gen2-kata-trustedlaunch` is ignored because it was skipped in that build. Same reason for the 3 ws skus - not used in Linux weekly release_)

downloading releaseNotes 'vhd-release-notes-marinerv2-gen2' from build '78770231'   
downloading releaseNotes 'vhd-release-notes-marinerv2-gen2-kata' from build '78770231'
downloading releaseNotes 'vhd-release-notes-marinerv2-gen2-fips' from build '78770231'
downloading releaseNotes 'vhd-release-notes-2004-cvm-gen2-containerd' from build '78770231'
downloading releaseNotes 'vhd-release-notes-2204-tl-gen2-containerd' from build '78770231'
downloading releaseNotes 'vhd-release-notes-2204-arm64-gen2-containerd' from build '78770231'
downloading releaseNotes 'vhd-release-notes-marinerv1' from build '78770231'        
downloading releaseNotes 'vhd-release-notes-1804-gen2-gpu-containerd' from build '78770231'
downloading releaseNotes 'vhd-release-notes-1804-containerd' from build '78770231'  
downloading releaseNotes 'vhd-release-notes-1804-gpu-containerd' from build '78770231'
downloading releaseNotes 'vhd-release-notes-marinerv2-gen1-fips' from build '78770231'
downloading releaseNotes 'vhd-release-notes-marinerv2-gen2-trustedlaunch' from build '78770231'
downloading releaseNotes 'vhd-release-notes-2204-gen2-containerd' from build '78770231'
downloading releaseNotes 'vhd-release-notes-1804-gen2-containerd' from build '78770231'
downloading releaseNotes 'vhd-release-notes-2004-fips-gen2-containerd' from build '78770231'
downloading releaseNotes 'vhd-release-notes-2204-containerd' from build '78770231'
downloading releaseNotes 'vhd-release-notes-marinerv2-gen2-arm64' from build '78770231'
downloading releaseNotes 'vhd-release-notes-1804-fips-containerd' from build '78770231'
downloading releaseNotes 'vhd-release-notes-marinerv1-gen2' from build '78770231'
downloading releaseNotes 'vhd-release-notes-1804-fips-gen2-containerd' from build '78770231'